### PR TITLE
update to scs-2.1.3

### PR DESCRIPTION
**Pull request by kalmarek**

### CODE REVIEW

**Comment by giordano** [comment]
Uhm, this needs a bit of care: we recently switched to enable ILP64 in openblas for aarch64, too.  This means we need to specify a compat version for OpenBLAS (and consequently Julia :persevere:)

**Comment by kalmarek** [comment]
* do we need to put `julia_compat="<1.7"` into build_tarball?
* is `OpenBlas_jll-0.3.13-2` the last version we can use?

**Comment by giordano** [comment]
> do we need to put julia_compat="<1.7" into build_tarball?

I think it's more important to specify the compat with OpenBLAS, the one for julia will be automatic

> is OpenBlas_jll-0.3.13-2 the last version we can use?

You can't really pick a specific revision, so you should do something like

```julia
Dependency("OpenBLAS_jll", v"0.3.10"; compat="=0.3.10")
```

The first version is the one used to compile, `compat` is the compat bound that will be used in the project file.  It'd be good to leave a comment that this is necessary because of the change to aarch64 OpenBLAS.

File: S/SCS/build_tarballs.jl [hunk 1]
```diff
@@ -40,7 +40,10 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("OpenBLAS_jll")
+#     OpenBLAS_jll-0.3.13-3 opted into using ILP64 on aarch64
+#     (see https://github.com/JuliaPackaging/Yggdrasil/pull/2590)
+#     but we still try to compile with `-lopenblas` there.
+    Dependency("OpenBLAS_jll", v"0.3.12", compat="<0.3.13")
```
Comment (giordano) [inline S/SCS/build_tarballs.jl:46]: No Julia version uses 0.3.12, only 0.3.10 (julia v1.6) and 0.3.13 (julia v1.7 at the moment)

Comment (giordano) [reply S/SCS/build_tarballs.jl:46]: But it shouldn't really matter for linking purposes.

**Comment by kalmarek** [comment]
ok, thanks for help!

**Pull request merged**